tests/operators/test_python.py

```diff
@@ -311,6 +312,314 @@ def func(**context):
         python_operator.run(start_date=DEFAULT_DATE, end_date=DEFAULT_DATE)
 
 
+class TestAirflowTask(TestPythonBase):
```

Comment:
In case of tests I am personally in favour of repeating code than creating dependencies between tests.